src/Vault.sol

```diff
@@ -62,8 +62,8 @@ contract Vault {
     }
 
     struct VaultInfo {
-        uint16 collThres; //2 decimals precision (factor 100)
-        uint8 liqThres; //2 decimals precision (factor 100)
+        uint16 collFactor; //2 decimals precision (factor 100)
```

Comment:
I thought we said collFactor should not be stored anymore?

Reply:
Yes true, it will be done in another pr since this one is already big